src/components/CustomTable.tsx

```
@@ -1,41 +1,188 @@
 
 type HeaderType = {
-    name: string,
-    id: string
+  name: string;
+  id: string;
+};
+
+type CustomTable = {
+  header: HeaderType[]; 
+  data: UsageData[] | null, 
+  rowUpdate?: (row:UsageData) => void  | undefined 
```

Biome (1.9.4)

[error] void is confusing inside a union type.

Unsafe fix: Use undefined instead.


(lint/suspicious/noConfusingVoidType)